internal/watch/watcher_darwin.go

```diff
@@ -38,7 +38,7 @@ func (d *darwinNotify) loop() {
 				e.Path = filepath.Join("/", e.Path)
 
 				_, isPathWereWatching := d.pathsWereWatching[e.Path]
-				if e.Flags&fsevents.ItemIsDir == fsevents.ItemIsDir && e.Flags&fsevents.ItemCreated == fsevents.ItemCreated && isPathWereWatching {
+				if e.Flags&fsevents.ItemIsDir == fsevents.ItemIsDir && isPathWereWatching {
 					// This is the first create for the path that we're watching. We always get exactly one of these
 					// even after we get the HistoryDone event. Skip it.
```

Comment:
nit: this comment is misleading now, I'd delete it

Reply:
updated!